app/styles/headers/_mobile-second.scss

```diff
@@ -0,0 +1,66 @@
+// mobile styles for second header
+
+@media (max-width: 550px) { 
+	.header-second {
+		padding: 10px 0px;
+		font-weight: 400;
+		font-size: 16px;
+		width: 100%;
+		max-height: 300px;
+}
+
+	.header-second-content{
+		max-height: 360px;
+
+	}
+	.header-second__bars{
+		right: 3px;
+		font-size: 24px;
+		top: 20px;
+		position: absolute;
+
+	}
+	.header-second__nav{
+		display: none;
+
+	}
+
+	.header-second__check:checked + .header-second__nav{
+		display: flex;
+		flex-flow: column wrap;
+		align-items: center;
+		margin-top: 70px;
+		width: 100%;
+		position: absolute;
+		z-index: 99;
```

Comment:
please, don't use such big amounts of `z-index`. Set it to 1. If you've already had block with `z-index: 1` and you need to place it below another one, set `z-index: 2`. But it is unnecessary to set 99

Reply:
ok, I fixed